dispatcher/process.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+import multiprocessing
+from typing import Iterable, Optional, Union
+
+from dispatcher.worker.task import work_loop
+
+
+class ProcessProxy:
+    def __init__(self, args: Iterable, finished_queue: multiprocessing.Queue) -> None:
```

Comment:
The args type is incomplete. Would this be more accurate?
```suggestion
    def __init__(self, args: Iterable[int | str], finished_queue: multiprocessing.Queue) -> None:
```

Reply:
Looks good, included.